src/main/java/io/stargate/sgv2/jsonapi/config/OperationsConfig.java

```diff
@@ -109,4 +113,54 @@ interface LwtConfig {
     @WithDefault("3")
     int retries();
   }
+
+  /** Cassandra/AstraDB related configurations. */
+  @NotNull
+  @Valid
+  DatabaseConfig databaseConfig();
+
+  interface DatabaseConfig {
+
+    /** Database type can be <code>cassandra</code> or <code>astra</code>. */
+    @WithDefault(CASSANDRA)
+    String type();
+
+    /** Username when connecting to cassandra database (when type is <code>cassandra</code>) */
+    @Nullable
+    @WithDefault("cassandra")
+    String userName();
+
+    /** Password when connecting to cassandra database (when type is <code>cassandra</code>) */
+    @Nullable
+    @WithDefault("cassandra")
+    String password();
+
+    /** Fixed Token used for Integration Test authentication */
+    @Nullable
+    @WithDefault("not in tests")
+    String fixedToken();
+
+    /** Cassandra contact points (when type is <code>cassandra</code>) */
+    @Nullable
+    @WithDefault("127.0.0.1")
+    List<String> cassandraEndPoints();
+
+    /** Cassandra contact points (when type is <code>cassandra</code>) */
+    @Nullable
+    @WithDefault("9042")
+    int cassandraPort();
+
+    /** Local datacenter that the driver must be configured with */
+    @NotNull
+    @WithDefault("datacenter1")
+    String localDatacenter();
+
+    /** Time to live for CQLSession in cache in seconds. */
+    @WithDefault("300")
```

Comment:
Are we sure we really want 5 minutes over 1 -- this is not time-to-live for connection (AFAIK) but time unused connection remains for reuse after last call. That is, idle wait.

Reply:
When session is evicted from cache, the session will be closed. Changed to 300 as per yesterday's conversation.

Reply:
Also these values can be override using environment value in the chart.